src/components/InactiveClientsList.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import { useHistory } from 'react-router-dom';
+import { useQuery } from '@apollo/client';
+import { Grid } from '@material-ui/core'
+import { orderBy } from 'lodash'
+
+import ClientTile from './ClientTile'
+import LoadingProgress from './LoadingProgress'
+import {GET_INACTIVE_CLIENTS, GET_INACTIVE_CLIENTS_COUNT} from '../operations/queries/ClientQueries'
+import FormGroup from '@material-ui/core/FormGroup'
+import FormControlLabel from '@material-ui/core/FormControlLabel'
+import Switch from '@material-ui/core/Switch'
+import Box from '@material-ui/core/Box'
+
+const InactiveClientsList = (props) => {
+    const history = useHistory()
+    const { loading, error, data } = useQuery(GET_INACTIVE_CLIENTS);
```

Comment:
Since there are 2 sets of `loading/error/data` maybe it would be better to be explicit in both places

```
{
    loading: loadingInactiveClients,
    error: errorInactiveClients,
    data: dataInactiveClients,
}
{
    loading: loadingInactiveClientsCount,
    error: errorInactiveClientsCount,
    data: dataInactiveClientsCount,
}
```